reference/mongodb/mongodb/driver/writeconcern/getwtimeout.xml

```diff
@@ -49,10 +49,8 @@
       <row>
        <entry>PECL mongodb 1.7.0</entry>
        <entry>
-        On 32-bit systems, this method will return a <classname>MongoDB\BSON\Int64</classname>
-        instance if the WriteConcern object was created with a <literal>wTimeout</literal>
-        that exceeds the 32-bit range. On 64-bit systems, this method will
-        always return an integer value.
+        On 32-bit systems, this method will always truncate the <literal>wTimeout</literal>
+        value if it exceeds the 32-bit range.
```

Comment:
Did this method _ever_ return an Int64 object? I think the most recent, relevant issue pertaining to this behavior is [PHPC-1411](https://jira.mongodb.org/browse/PHPC-1411). Should we note that it emits a warning when truncating?

Reply:
Good point, I documented the warning accordingly.